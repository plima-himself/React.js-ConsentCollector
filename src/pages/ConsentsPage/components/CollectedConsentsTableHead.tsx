import React from "react";
import { TableCell, TableHead, TableRow } from "@mui/material";

const CollectedConsentsTableHead: React.FC = () => {
  return (
    <TableHead>
      <TableRow>
        <TableCell sx={{ width: "20%" }}>
          <strong>Name</strong>
        </TableCell>

        <TableCell sx={{ width: "25%" }}>
          <strong>Email</strong>
        </TableCell>

        <TableCell sx={{ width: "55%" }}>
          <strong>Consent given for</strong>
        </TableCell>
      </TableRow>
    </TableHead>
  );
};

export default CollectedConsentsTableHead;
