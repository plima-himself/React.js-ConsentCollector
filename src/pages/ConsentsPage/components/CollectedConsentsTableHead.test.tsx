import { render } from "@testing-library/react";
import CollectedConsentsTableHead from "./CollectedConsentsTableHead";

describe("CollectedConsentsTableHead", () => {
  // Snapshot test to ensure table headers render as expected.
  // This helps catch unintentional changes in the structure or content of the table head.
  it("matches snapshot", () => {
    const { container } = render(
      <table>
        <CollectedConsentsTableHead />
      </table>
    );

    expect(container).toMatchSnapshot();
  });
});
