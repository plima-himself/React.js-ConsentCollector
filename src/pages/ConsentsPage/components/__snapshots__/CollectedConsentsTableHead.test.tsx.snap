// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CollectedConsentsTableHead > matches snapshot 1`] = `
<div>
  <table>
    <thead
      class="MuiTableHead-root css-1a7iywq-MuiTableHead-root"
    >
      <tr
        class="MuiTableRow-root MuiTableRow-head css-1xtozoh-MuiTableRow-root"
      >
        <th
          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-6ssiqt-MuiTableCell-root"
          scope="col"
        >
          <strong>
            Name
          </strong>
        </th>
        <th
          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-n5qgyl-MuiTableCell-root"
          scope="col"
        >
          <strong>
            Email
          </strong>
        </th>
        <th
          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-en18vu-MuiTableCell-root"
          scope="col"
        >
          <strong>
            Consent given for
          </strong>
        </th>
      </tr>
    </thead>
  </table>
</div>
`;
